osquery/tables/system/posix/augeas.cpp

```diff
@@ -168,12 +182,27 @@ QueryData genAugeas(QueryContext& context) {
     patterns.insert(nodes.begin(), nodes.end());
   }
 
+  if (context.hasConstraint("node", LIKE)) {
+    auto nodes = context.constraints["node"].getAll(LIKE);
+    for (std::string node : nodes) {
```

Comment:
I wonder if you can make this slightly more performant with `std::string&`

Reply:
You could also have `convertWildcards` return a new string and write `patters.insert(convertWildcards(node));`